src/models/constructor-items/blocks.ts

```diff
@@ -376,6 +378,35 @@ export interface ShareBlockProps {
     title?: string;
 }
 
+export enum FormBlockDataTypes {
+    YANDEX = 'yandex',
+    HUBSPOT_INLINE = 'hubspot-inline',
```

Comment:
can we do that if this is not major?
```suggestion
    HUBSPOT = 'hubspot',
```